modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -884,35 +883,30 @@ definition {
 					key_objectLabel = "${objectLabel}",
 					locator1 = "ObjectAdmin#CHOOSE_OBJECT_LABEL");
 
-				Click(
-					key_inputMethod = "${inputMethod}",
-					locator1 = "ObjectAdmin#INPUT_METHOD");
+				if ("${inputMethod}" == "true") {
+					Click(locator1 = "ObjectAction#INPUT_METHOD");
 
-				Type(
-					locator1 = "ObjectAction#INPUT_VALUE",
-					value1 = "${inputValue}");
-			}
-		}
+					Type(
+						locator1 = "ObjectAction#INPUT_VALUE",
+						value1 = "${inputValue}");
+				}
 
-		if (isSet(inputExpression)) {
-			if ("${inputCondition}" == "true") {
-				ObjectAdmin.enableCondition();
+				else if("${inputMethod}" == "false") {
+					Type(
+						locator1 = "ObjectAction#INPUT_NEW_VALUE",
+						value1 = "${inputExpression}");
 
-				Type(
-					locator1 = "ObjectAction#INPUT_EXPRESSION",
-					value1 = "${inputExpression}");
-			}
-			else if ("${inputCondition}" == "false") {
-				ObjectAdmin.disableCondition();
+				}
+				
 			}
 		}
 
 		if (isSet(inputExpression)) {
-			if ("${inputCondition}" == "true") {
-				Type(
-					locator1 = "ObjectAction#INPUT_VALUE",
-					value1 = "${inputExpression}");
-			}
+			ObjectAdmin.enableCondition();
+
+			Type(
+				locator1 = "ObjectAction#INPUT_EXPRESSION",
+				value1 = "${inputExpression}");
 		}
```

Comment:
Honestly, it's a lot cleaner and better for the original author to fix mistakes like this. Good job taking care of this though.